near-sdk-macros/src/core_impl/code_generator/snapshots/callback_args_vec.snap

```diff
@@ -30,18 +36,24 @@ pub extern "C" fn method() {
                     ::near_sdk::PromiseResult::Successful(x) => x,
                     _ => {
                         ::near_sdk::env::panic_str(
-                            &::std::format!(
-                                "Callback computation {} was not successful", i
+                            concat!(
+                                "Callback computation ", stringify!(i),
```

Comment:
`stringify!(i)` will return `"i"` instead of the value under `i` variable, which means that the error will be `Callback computation i was not successful`, instead of `Callback computation 0 was not successful`.

I think `format!` is unavoidable here.